bluemira/codes/process/api.py

```diff
@@ -80,9 +79,42 @@ def __init__(self, filename):
     PROCESS_DICT = get_dicts()
 
 
-DEFAULT_INDAT = os.path.join(
-    get_bluemira_path("codes/process"), "PROCESS_DEFAULT_IN.DAT"
-)
+@dataclass
+class INVariable:
+    """
+    Process io.in_dat.INVariable replica
+
+    Used to simulate what process does to input variables
+    for the InDat input file writer
+    """
```

Comment:
Might be worth noting _why_ we need to simulate process in-variables, I remember from our conversations that this wasn't a straight-forward decision. Putting down some of that reasoning might stop has re-hashing the same conversations